docs/feature_dynamic_macros.md

```diff
@@ -6,11 +6,13 @@ You can store one or two macros and they may have a combined total of 128 keypre
 
 To enable them, first include `DYNAMIC_MACRO_ENABLE = yes` in your `rules.mk`. Then, add the following keys to your keymap:
 
-* `DYN_REC_START1` — start recording the macro 1,
-* `DYN_REC_START2` — start recording the macro 2,
-* `DYN_MACRO_PLAY1` — replay the macro 1,
-* `DYN_MACRO_PLAY2` — replay the macro 2,
-* `DYN_REC_STOP` — finish the macro that is currently being recorded.
+|Key               |Alias     |Description                                        |
+|------------------|----------|---------------------------------------------------|
+|`DYN_REC_START1`  |`DM_REC1` |Start recording Macro 1                            |
```

Comment:
Could you add this to `keycodes.md` as well?

Reply:
Oops. :rofl:

Reply:
I think I'll do this in another PR.

Reply:
Pushed to the wrong branch? 😄

Reply:
@noroadsleft push it to my branch again :D

Reply:
@fauxpark Forgot that they needed to go into the keycodes Doc as well.

@drashna K, will do.